app/scripts/modules/core/src/pipeline/status/ArtifactList.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { IArtifact } from 'core/domain';
+
+import './artifactList.less';
+
+export interface IArtifactListProps {
+  artifacts: IArtifact[],
+};
+
+export interface IArtifactListState {}
+
+export class ArtifactList extends React.Component<IArtifactListProps, IArtifactListState> {
+  constructor(props: IArtifactListProps) {
+    super(props);
+  }
+
+  private tooltip(artifact: IArtifact): string {
+    const tooltipEntries = [];
+    if (artifact.name) {
+      tooltipEntries.push(`Artifact: ${artifact.name}`);
```

Comment:
I think it might be safer to reference this as `Name: ${artifact.name}`, or `Artifact name: ...` if that feels better

Reply:
:+1: